src/zabapgit_repo_impl.prog.abap

```diff
@@ -500,19 +500,21 @@ CLASS lcl_repo IMPLEMENTATION.
 
   ENDMETHOD.
 
-  METHOD get_files_local.
+   METHOD get_files_local.
```

Comment:
please move "METHOD" 1 character to the left, also see error in travis from abaplint, https://travis-ci.org/larshp/abapGit/builds/193795985  (not very user friendly)